pages/api/contact.js

```diff
@@ -0,0 +1,33 @@
+import { Resend } from 'resend'
+import ContactEmailTemplate from '../../components/emails/contactEmailTemplate'
+
+const resend = new Resend(process.env.RESEND_API_KEY)
+
+export default async (req, res) => {
+  if (req.method !== 'POST') {
+    return res.status(405).json({ error: 'Method not allowed' })
+  }
+
+  const { name, email, subject, message } = req.body
+  if (!name || !email || !subject || !message) {
+    return res.status(400).json({ error: 'All fields are required' })
+  }
+
+  const { data, error } = await resend.emails.send({
+    from: 'No Reply <noreply@theteacherfund.com>',
+    to: 'joel@theteacherfund.com',
+    subject,
+    react: ContactEmailTemplate({
+      name,
+      email,
+      subject,
+      message
```

Comment:
let's add cc: email to this list. looks like it's a top level key/value pair for the resend.send() method.

https://resend.com/docs/api-reference/emails/send-email